google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClientTests.java

```diff
@@ -683,6 +693,78 @@ public void testListBackups() {
     assertThat(actualResults).containsExactlyElementsIn(expectedResults);
   }
 
+  @Test
+  public void testCopyBackup() {
+    // Setup
+    Mockito.when(mockStub.copyBackupOperationCallable())
+        .thenReturn(mockCopyBackupOperationCallable);
+
+    Timestamp startTime = Timestamp.newBuilder().setSeconds(1234).build();
+    Timestamp endTime = Timestamp.newBuilder().setSeconds(5678).build();
+
+    // Use existing adminClient as destination project:
+    String dstProjectId = PROJECT_ID;
```

Comment:
nit - I would just use `PROJECT_ID` and `INSTANCE_ID` inline and remove these local variables.

Reply:
Done.